src/pages/signup.js

```diff
@@ -32,12 +34,14 @@ const SignUp = () => {
   const emailInputRef = useRef(null)
   const passwordInputRef = useRef(null)
 
-  const [form, setForm] = useState({email: "",
+  const [form, setForm] = useState({
```

Comment:
Codacy found an issue: [Unexpected line break after this opening brace.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=6000161)